.github/workflows/test.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile --ignore-scripts
       - name: Run unit tests
-        run: ./node_modules/.bin/jest
+        run: pnpm test:short
```

Comment:
Can you rename it to `pnpm unit` for consistency with my other open source projects?